README.md

```diff
@@ -41,8 +41,4 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly add a license file.
- Please, kindly update the author header by adding the names of the author that has collaborated on this project.
- Please, kindly remove the headers that are not in use to make your readme more professional.
- Update the getting started headers by telling the user how to have your project on their local machine.
- Kindly update the acknowledgment header by acknowledging microverse.
- Kindly follow the [microverse readme template](https://github.com/microverseinc/readme-template) to restructure your readme so it conforms to the template format arrangement.